samples/notebooks/image-captioning-gcp/README.md

```diff
@@ -0,0 +1,40 @@
+# Image Captioning TF 2.0
+
+## About
+This notebook is an example of how to convert an existing Tensorflow notebook into a Kubeflow pipeline using jupyter notebook.  Specifically, this notebook takes an example tensorflow notebook, [image captioning with attention](https://colab.sandbox.google.com/github/tensorflow/docs/blob/master/site/en/r2/tutorials/text/image_captioning.ipynb), and creates a kubeflow pipeline.  This pipeline produces a model that can generate captions for images.
+
+### Example generated captions
+The following example captions were created when using `num_examples = 30000`.
+
+![bus-output](https://user-images.githubusercontent.com/17008638/61419442-17989a00-a8b3-11e9-9ab3-a5a304ff96d0.PNG)
+
+![sandwich-output](https://user-images.githubusercontent.com/17008638/61419487-44e54800-a8b3-11e9-9b7f-68ccc970c10d.PNG)
+
+## Setup
+
+### Setup notebook server
+This pipeline requires you to [setup a notebook server](https://www.kubeflow.org/docs/notebooks/setup/) in the Kubeflow UI.  After you are setup, upload this notebook and then run it in the notebook server.
+
+### Create a GCS bucket
+This pipeline requires a GCS bucket.  If you haven't already, [create a GCS bucket](https://cloud.google.com/storage/docs/creating-buckets) to run the notebook.  Make sure to create the storage bucket in the same project that you are running Kubeflow on to have the proper permissions by default.  You can also create a GCS bucket by running `gsutil mb -p <project_name> gs://<bucket_name>`.
+
+### Upload the notebook in the Kubeflow UI
+In order to run this pipeline, make sure to upload the notebook to your notebook server in the Kubeflow UI.  You can clone this repo in the Jupyter notebook server by connecting to the notebook server and then selecting New > Terminal.  In the terminal type `git clone https://github.com/kubeflow/pipelines.git`.
+
+### Download dataset
```

Comment:
Could we add the docker-build-the-base-image command in this Setup section? Instead of hosting the image and maintaining it, maybe we can have it as a setup step. WDYT?

Reply:
Yeah that works, I'll put the instructions for uploading the docker image in the setup instructions.

Reply:
Thanks